### PR TITLE
US108829: adding incidents issue type and changing some text

### PR DESCRIPTION
**Pull request by msanchezal**
https://rally1.rallydev.com/#/236117588196d/detail/userstory/341575783452

### CODE REVIEW

File: src/main/resources/cloud-insight-for-jira.properties [hunk 1]
```diff
@@ -38,17 +38,20 @@ ci.atlassianplugin.boostrap.msg.connectionerror =An error occurred during connec
...
+ci.constant.scheme.description = Alert Logic Integration
+ci.constant.issuetypescreen.name = Alert Logic Screen Scheme
 ci.constant.issuetypescreen.description = Cloud Insight Issue Type
-ci.constant.issuetype.description =A Cloud Insight remediation step that contains information about the exposure to be addressed. If you properly disposed the remediation, the remediation step does not appear after the next scan.
+ci.constant.issuetype.description =A remediation step that contains information about the exposure to be addressed. If you properly disposed the remediation, the remediation step does not appear after the next scan.
 ci.constant.issuetype.name =Cloud Insight Remediation
-ci.constant.issuetypeschema.name =Cloud Insight Remediation Issue Type Scheme
-ci.constant.issuetypeschema.description =CI Remediation Issue Type Scheme
+ci.constant.issuetype.incident.name =Alert logic Incident
+ci.constant.issuetype.incident.description =An Alert logic incident type
```
Comment (scottarver) [inline src/main/resources/cloud-insight-for-jira.properties]: Alert logic should be "Alert Logic", both caps

Comment (msanchezal) [reply src/main/resources/cloud-insight-for-jira.properties]: thanks Jason, done

File: src/main/resources/cloud-insight-for-jira.properties [hunk 2]
```diff
@@ -38,17 +38,20 @@ ci.atlassianplugin.boostrap.msg.connectionerror =An error occurred during connec
...
+ci.constant.issuetypescreen.name = Alert Logic Screen Scheme
 ci.constant.issuetypescreen.description = Cloud Insight Issue Type
-ci.constant.issuetype.description =A Cloud Insight remediation step that contains information about the exposure to be addressed. If you properly disposed the remediation, the remediation step does not appear after the next scan.
+ci.constant.issuetype.description =A remediation step that contains information about the exposure to be addressed. If you properly disposed the remediation, the remediation step does not appear after the next scan.
 ci.constant.issuetype.name =Cloud Insight Remediation
-ci.constant.issuetypeschema.name =Cloud Insight Remediation Issue Type Scheme
-ci.constant.issuetypeschema.description =CI Remediation Issue Type Scheme
+ci.constant.issuetype.incident.name =Alert Logic Incident
+ci.constant.issuetype.incident.description =An Alert Logic Incident
+ci.constant.issuetypeschema.name =Alert logic Issue Type Scheme
```
Comment (scottarver) [inline src/main/resources/cloud-insight-for-jira.properties]:
```suggestion
ci.constant.issuetypeschema.name =Alert Logic Issue Type Scheme
```

File: src/main/resources/cloud-insight-for-jira.properties [hunk 3]
```diff
@@ -38,17 +38,20 @@ ci.atlassianplugin.boostrap.msg.connectionerror =An error occurred during connec
...
+ci.constant.issuetypeschema.description =Alert logic Issue Type Scheme
```
Comment (scottarver) [inline src/main/resources/cloud-insight-for-jira.properties]:
```suggestion
ci.constant.issuetypeschema.description =Alert Logic Issue Type Scheme
```

**Review by juankremer — APPROVED** [review]
+1

**Pull request merged**